File: ConsentCollector/ConsentCollector.Entities/Consent/SurveyQuestion.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ConsentCollector.Entities.Consent
{
    public sealed class SurveyQuestion : Entity
    {

        public SurveyQuestion(Guid IdSurvey, Guid IdQuestion)
        {
            this.IdSurvey = IdSurvey;
            this.IdQuestion = IdQuestion;
        }

        public Guid IdSurvey { get; set; }
        public Guid IdQuestion { get; set; }
        public Question Question { get; set; }
        public Survey Survey { get; set; }
    }
}

File: Web/src/app/models/surveyQuestion.ts
export class SurveyQuestion{
  id?: string;
  idSurvey:string;
  idQuestion:string;
  public constructor(idSurvey:string, idQuestion:string){
    this.idQuestion=idQuestion;
    this.idSurvey=idSurvey;
  }
}
